test/options/sourceMap.test.js

```diff
@@ -47,7 +47,7 @@ describe('Options', () => {
       const { source } = stats.toJson().modules[1]
 
       expect(source).toEqual(
-        'module.exports = "a { color: rgba(255, 0, 0, 1.0) }\\n\\n/*# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbInRlc3QvZml4dHVyZXMvY3NzL3N0eWxlLmNzcyJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUFBQSxJQUFJLDJCQUFZLEVBQUUiLCJmaWxlIjoidGVzdC9maXh0dXJlcy9jc3Mvc3R5bGUuY3NzIiwic291cmNlc0NvbnRlbnQiOlsiYSB7IGNvbG9yOiBibGFjayB9XG4iXX0= */"'
+        'module.exports = "a { color: rgba(255, 0, 0, 1.0) }\\n\\n/*# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbInRlc3QvZml4dHVyZXMvY3NzL3N0eWxlLmNzcyJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUFBQSxJQUFJLDRCQUFhIiwiZmlsZSI6InRlc3QvZml4dHVyZXMvY3NzL3N0eWxlLmNzcyIsInNvdXJjZXNDb250ZW50IjpbImEgeyBjb2xvcjogYmxhY2sgfVxuIl19 */"'
```

Comment:
/cc @jquense It is strange, why source map was changed

Reply:
i don't know, this was broken before my PR I think: https://github.com/postcss/postcss-loader/pull/425#issuecomment-584228103

Reply:
@jquense maybe postcss have bugs with source maps when we share AST?

Reply:
no these tests are failing on master right now

Reply:
the source map difference is unrelated to this PR, my guess is it's a difference from node 10 to 12, this repo is really inactive so maybe no one has noticed until now

Reply:
I think it is bug on postcss side when we share AST

Reply:
I faced with same problem in css-loader and put it in my TODO for the next major release

Reply:
I really don't think it's a bug with this PR, if I make NO changes to the code on a fresh checkout of postcss-loader/master these two tests fail.

Reply:
see: https://github.com/postcss/postcss-loader/pull/427

Reply:
Yep, let's merge, improving source maps for CSS pipeline in my TODO, so let's check and tests all in other PRs